test/refactoring/AddCodableImplementation/Outputs/enum/codable.swift.expected

```diff
@@ -18,10 +18,12 @@ enum Payload: Codable {
   init(from decoder: Decoder) throws {
     let container = try decoder.container(keyedBy: Payload.CodingKeys.self)
 
-    guard container.allKeys.count == 1 else {
+    var allKeys = ArraySlice(container.allKeys)
+
+    guard let onlyKey = allKeys.popFirst(), allKeys.isEmpty else {
```

Comment:
Perfect!

I was confused at first seeing `allKeys.isEmpty`. Then I realized it's mutating `allKeys` with the `popFirst()` (I initially just read it as _accessing_ the first key, not removing it from the array). 

Seems like a much more efficient (and safe) way for checking that's there's only 1 key.